### ISSUE TITLE
Feature request: Add the ability to hide source port, destination port or both

### ISSUE THREAD

**Issue by contentfree** [iptraf-ng/iptraf-ng#37]
This would be useful when trying to gauge what services are using a lot bandwidth by showing just the destination port (or just the IP address if both source and destination ports were hidden).

**Comment by vsamel** [comment]
In what module? In "IP traffic monitor"? You could try ""Statistical breakdowns --> By TCP/UDP port" to get an eye on ports usage. Isn't that sufficient?

**Comment by contentfree** [comment]
I'm thinking the opposite of that (but yes for IP traffic monitoring). I'm hoping to group all traffic going to a load-balanced domain (AWS global accelerator) into one line.On Dec 25, 2023, at 03:16, Vitezslav Samel ***@***.***> wrote:﻿
In what module? In "IP traffic monitor"? You could try ""Statistical breakdowns --> By TCP/UDP port" to get an eye on ports usage. Isn't that sufficient?

**Comment by vsamel** [comment]
So instead we are talking about grouping:
- group flows by the same destination address (all flows with the same destination address independent of destination port and source address and source port wil be accounted in one line)
- group flows by the same destination port (all flows with the same destination port independent of destination address and source address and source port wil be accounted in one line)
- group flows by the same source address and destination address (all flows with the same source address and the same destination address independent of destination port and source port wil be accounted in one line)
- group flows by the same source address and destination port (all flows with the same source address and the same destination port independent of destination address and source port wil be accounted in one line)
- etc.

Is any of this what you expect?

**Comment by contentfree** [comment]
Kinda. I think those are all available currently in the various views. What's missing is grouping by DNS names (traffic to multiple IP addresses that are at the same domain (like a set of load balancers)On Jan 9, 2024, at 10:45, Vitezslav Samel ***@***.***> wrote:﻿
So instead we are talking about grouping:

group flows by the same destination address (all flows with the same destination address independent of destination port and source address and source port wil be accounted in one line)
group flows by the same destination port (all flows with the same destination port independent of destination address and source address and source port wil be accounted in one line)
group flows by the same source address and destination address (all flows with the same source address and the same destination address independent of destination port and source port wil be accounted in one line)
group flows by the same source address and destination port (all flows with the same source address and the same destination port independent of destination address and source port wil be accounted in one line)
etc.

Is any of this what you expect?

**Comment by vsamel** [comment]
IMHO dependency on DNS names is too much fragile to be done correctly and also will be inefficient. Sorry I won't go this way.

**Comment by contentfree** [comment]
I agree, most of the time this is true. But it would be supremely useful to see all traffic going from my server to a load balancer - which relies on DNS resolution - grouped together in a single item.Pretty please? On Jan 11, 2024, at 06:02, Vitezslav Samel ***@***.***> wrote:﻿
IMHO dependency on DNS names is too much fragile to be done correctly and also will be inefficient. Sorry I won't go this way.
